### DISCUSSION TITLE
Can only see the first word of the title when I compose a new discussion

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2825] [Discussions]
Here we see the title box deserves it's own line. Otherwise the user can only see one word of the title when they are entering it. There has never been such a small title box in the history of Internet title boxes, I'm sure.
![Screenshot_20201202-221402_1](https://user-images.githubusercontent.com/1230959/100884534-8dbf9000-34ec-11eb-93de-c2dd55e8ad8c.jpg)
In the image, the title box is the box that has the words "Copying..." in it.

**Comment by ghost** [comment]
Maybe when making a discussion on mobile the category box could shrink to just the emoji when you are not using it and focused on the title so the title has more room. I guess they could also just make it two lines... But, that’s just an idea for the GitHub team, maybe they could do something like that to fix it.
